feat(ARCH-314): add cdn build script and webpack injector (#97)
Talend/ui-scripts: cdn/build-cdn/README.md

> # Build cdn script
> 
> Let's download umds on https://unpkg.com !
> 
> Context:
> 
> On one hand we have `@talend/module-to-cdn` package which contains the configuration of available umds from the community.
> 
> On the other hand we have `Talend/ui/versions/dependencies.json` file which contains the minimal version required in our cross projects dependencies.
> 
> This script take those two into account to download all possible libraries
> 
> # How to use
> 
> ```
> /index.js -d -e ';@angular;' --versions ../../../ui/versions/dependencies.json
> ```
> 
> options:
> 
> - -d debug will print many information
> - -e to exclude some package, scope. Values separated by ';'
> - --versions to give the path to a minimal versions requirement. here you should use the ui dependencies.json
> - --dev to also download umd for dev mode. By default the script only download minified umds.

Reviewer: ```suggestion
# Build CDN script

Let's download UMD files from https://unpkg.com!

## Context

On one hand, we have the `@talend/module-to-cdn` package which contains the configuration of available UMD files from the community.

On the other hand, we have the `Talend/ui/versions/dependencies.json` file which contains the minimal version required in our cross projects dependencies.

This script takes those two into account to download all possible libraries

# How to use

```bash
$> ./index.js -d -e ';@angular;' --versions ../../../ui/versions/dependencies.json
```

## options

- `-d` debug will print much information
- `-e` to exclude some package, scope. Values separated by ';'
- `--versions` to give the path to a minimal version requirement. here you should use the Talend/UI dependencies.json
- `--dev` to also download the UMD file for dev mode. By default, the script only downloads minified UMD files.
```
